core/templates/dev/head/components/create_button/create_activity_button_directive.html

```diff
@@ -1,7 +1,7 @@
 <div class="oppia-navbar-button-container">
   <!-- User must complete the registration process. -->
   <button class="btn oppia-navbar-button oppia-navbar-hide-on-small-width protractor-test-create-activity oppia-transition-200"
-          ng-click="onRedirectToLogin('/creator_dashboard?mode=create')" ng-if="!userIsLoggedIn">
+          ng-click="onRedirectToLogin('/creator_dashboard?mode=create')" ng-if="userIsLoggedIn === false">
```

Comment:
Perhaps add comment saying why "=== false" rather than "!".

But actually perhaps "!" is fine if you use a default value? What are you plannign to set userIsLoggedIn to at the outset? Should make clear (in comments in the JS) what that value is intended to mean, conceptually.